Member-Travel-Fund.md

```diff
@@ -1,78 +1,2 @@
-
-**The following considerations are made for approval of the request:**
-
-* Impact: Preference given to underserved communities. More specifically,
-within a subject the foundation is trying to promote awareness about (e.g.
-general knowledge about the foundation) there are people, communities and
-geographies where that knowledge is not very widespread. This is where
-preference will be given. For instance, an event in San Francisco would be
-less attractive than an event of the same size in Kansas City.
-* Cost: The larger the stipend the more critically the travel fund admins will 
-consider the application. Budget for this program is a finite resource.
-* Equity: Preference given to individuals who do not have a corporate travel fund and have
-not previously received a stipend.
-
-Examples of prior travel requests include attending a TC39 meeting, Collab Summit, and Code + Learn mentoring.
-
-## 2017 Approved Travel Requests 
-  
-Name | Event | Kind of participation | Location | Date | Stipend size
----- | ----- | --------------------- | -------- | ---- | ------------
-Rich Trott | NINA 2016 | Collaborator Summit and Code & Learn | Austin, TX, US | 29 Nov – 2 Dec 2016 | US$ 2000
-Anna Henningsen | NINA 2016 | Collaborator Summit and Code & Learn | Austin, TX, US | 29 Nov – 2 Dec 2016 | US$ 2530
-Stephen Belanger | NINA 2016 | Collaborator Summit | Austin, TX, US | 1 Dec – 2 Dec 2016 | US$ 800
-Italo A. Casas | NINA 2016 | Collaborator Summit | Austin, TX, US | 1 Dec – 2 Dec 2016 | US$ 644
-Anna Henningsen | TC39 Meeting | Attendance | San Jose, CA, US | 24 Jan – 26 Jan 2017 | US$ 1048
-Michaël Zasso | Collaborator Summit | Attendance | Berlin, DE | 4 May – 5 May 2017 | US$ 318
-Anna Henningsen | Collaborator Summit | Attendance | Berlin, DE | 4 May – 5 May 2017 | US$ 240
-Santiago Gimeno | Collaborator Summit | Attendance | Berlin, DE | 4 May – 5 May 2017 | US$ 160
-Calvin Metcalf | Collaborator Summit | Attendance | Berlin, DE | 4 May – 5 May 2017 | US$ 600
-Никита Сковорода | Collaborator Summit | Attendance | Berlin, DE | 4 May – 5 May 2017 | €573
-Bryan English | JSConf.cn 2017 | Code & Learn Mentor | Shanghai, CN | 15 July - 16 July 2017 | $1100
-Tobias Nießen | NINA 2017 | Collaborator Summit and Code & Learn | Vancouver, BC, CA | 5 Oct - 8 Oct 2017 | €1300
-Tiancheng Gu | NINA 2017 | Collaborator Summit | Vancouver, BC, CA | 3 Oct – 8 Oct 2017 | US$ 1200
-Ruben Bridgewater | NINA 2017 | Collaborator Summit and Code & Learn | Vancouver, BC, CA | 4 Oct - 8 Oct 2017 | €1500
+The travel fund is now overseen in
```

Comment:
Non-blocking nit: Maybe `Travel fund policy and usage records have been moved to https://github.com/nodejs/admin/blob/master/MEMBER_TRAVEL_FUND.md.`